be/src/exec/csv_scanner.cpp

```diff
@@ -429,13 +429,13 @@ Status CSVScanner::_parse_csv(Chunk* chunk) {
 
         if (fields.size() != _num_fields_in_csv) {
             if (_counter->num_rows_filtered++ < REPORT_ERROR_MAX_NUMBER) {
-                std::string error_msg = make_column_count_not_matched_error_message(_num_fields_in_csv,
-                                                                                    row.columns.size(), _parse_options);
+                std::string error_msg =
```

Comment:
You need to add tests to cover this case.

Reply:
updated